src/index.test.ts

```diff
@@ -102,10 +107,12 @@ test('export', (t): void => {
           '@typescript-eslint/no-misused-new': 'error',
           '@typescript-eslint/no-misused-promises': 'error',
           '@typescript-eslint/no-namespace': 'error',
+          '@typescript-eslint/no-non-null-asserted-optional-chain': 'error',
```

Comment:
This won't do anything at the moment since `@typescript-eslint/no-non-null-assertion` makes it so that you can never come here. But it's still nice to enable it, since we might change `@typescript-eslint/no-non-null-assertion` in the future 👍